nncf/common/pruning/model_analysis.py

```diff
@@ -244,29 +254,34 @@ def __init__(self, graph: NNCFGraph,
 
         self._is_depthwise_conv_fn = is_depthwise_conv_fn
 
-        self.can_prune = {idx: True for idx in self.graph.get_all_node_idxs()}
-        self.accept_pruned_input = {idx: True for idx in self.graph.get_all_node_idxs()}
+        self.can_prune = {idx: True for idx in self.graph.get_all_node_ids()}
+        self.accept_pruned_input = {idx: True for idx in self.graph.get_all_node_ids()}
 
     def node_propagate_can_prune_attr(self, nncf_node: NNCFNode) -> bool:
         """
-        Whether node propagates can_prune attr through. That means a node can propagate pruning mask
-         (for example,  activations propagate mask, but convolutions stop mask propagation)
-        :param nncf_node: node to work with
-        :return: bool: propagates this node can_prune throw or not
+        Whether node propagates `can_prune` attr through. That means a node can propagate pruning mask
```

Comment:
Whether the node can propagate the `can_prune` attr

Reply:
Thanks! I fixed it.